botfront/imports/ui/components/templates/MetadataForm.jsx

```diff
@@ -25,66 +23,57 @@ class AutoFormMetadata extends AutoForm {
     }
 }
 
-function ResponseMetadataForm({
-    responseMetadata, onChange,
-}) {
-    const schema = buildASTSchema(parse(`
+
+export const basicSchemaString = `
         type ResponseMetadata {
             linkTarget: String!
             userInput:  String!
             forceOpen: Boolean!
             forceClose: Boolean!
-        }
+        }`;
 
-        # This is required by buildASTSchema
-        type Query { anything: ID }
-    `)).getType('ResponseMetadata');
-
-    const defaultModel = {
-        linkTarget: '_blank',
-        userInput: 'show',
-        forceOpen: false,
-        forceClose: false,
-        domHighlight: {},
-        customCss: {},
-        pageChangeCallbacks: null,
-    };
+export const defaultModel = {
+    linkTarget: '_blank',
+    userInput: 'show',
+    forceOpen: false,
+    forceClose: false,
+};
 
-    const schemaData = {
-        linkTarget: {
-            label: 'Where should the links open?',
-            defaultValue: '_blank',
-            allowedValues: ['_blank', '_self'],
-            options: [
-                { label: 'In the current tab', value: '_self' },
-                { label: 'In a new tab', value: '_blank' },
-            ],
-        },
-        userInput: {
-            label: 'How should the user input field be rendered?',
-            defaultValue: 'show',
-            allowedValues: ['show', 'hide', 'disable'],
-            options: [
-                { label: 'Show', value: 'show' },
-                { label: 'Hide', value: 'hide' },
-                { label: 'Disable', value: 'disable' },
-            ],
-        },
-        forceOpen: {
-            label: 'Force the chat widget to open? (Otherwise it will appear as a tooltip if the widget is closed)',
```

Comment:
would it be better if the label was "Force the chat widget to open? (The message will appear as a tooltip if the widget is closed)"